specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/ContainerAppsJobs.json

```diff
@@ -0,0 +1,783 @@
+      "required": [
+        "replicaTimeout",
+        "crashLoopRetryLimit",
+        "triggerType"
+      ],
+      "properties": {
+        "secrets": {
+          "description": "Collection of secrets used by a Container app",
+          "type": "array",
+          "items": {
+            "$ref": "./CommonDefinitions.json#/definitions/Secret"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ]
+        },
+        "triggerType": {
+          "description": "Type of the job - Schedule | Event | Manual",
+          "enum": [
+            "Scheduled",
+            "Event",
+            "Manual"
+          ],
+          "type": "string",
+          "default": "Single",
+          "x-ms-enum": {
+            "name": "TriggerType",
+            "modelAsString": true
+          }
+        },
+        "replicaTimeout": {
+          "description": "Number of seconds from the startTime for each replica after which the system terminate it.",
+          "format": "int32",
+          "type": "integer"
+        },
+        "replicaRetryLimit": {
+          "format": "int32",
+          "description": "Maximum number of replica retries before failing the whole job.",
+          "type": "integer"
+        },
+        "manualTriggerConfig": {
+          "type": "object",
+          "description": "Manual Trigger configurations. Properties replicaCount and parellelism would be set to 1 by default",
+          "properties": {
+            "replicaCount": {
+              "description": "Number of replicas that will be created when the Container App Job is triggered.",
+              "format": "int32",
+              "type": "integer"
+            },
+            "parallelism": {
+              "description": "Number of parallel jobs that can run at a given time.",
+              "format": "int32",
+              "type": "integer"
+            }
+          }
+        },
+        "scheduleTriggerConfig": {
+          "description": "Scheduled Trigger configurations. Properties completions and parellelism would be set to 1 by default",
+          "required": [
+            "cronExpression"
+          ],
+          "type": "object",
+          "properties": {
+            "replicaCount": {
+              "description": "Number of replicas that will be created when the Container App Job is triggered.",
+              "format": "int32",
+              "type": "integer"
+            },
+            "cronExpression": {
+              "description": "Cron formatted repeating schedule (\"* * * * *\")\nfor cronjobs.",
+              "type": "string"
+            },
+            "parallelism": {
+              "description": "Number of parallel jobs that can run at a given\ntime.",
+              "format": "int32",
+              "type": "integer"
+            }
+          }
+        },
+        "registries": {
+          "description": "Collection of private container registry credentials for containers used by the Container apps job",
```

Comment:
"Collection of private container registry credentials used by a Container apps job"